include/uv-win.h

```diff
@@ -647,3 +647,17 @@ RB_HEAD(uv_timer_tree_s, uv_timer_s);
 #ifndef X_OK
 #define X_OK 1
 #endif
+
+#if _MSC_VER >= 1900
+#undef stdin
```

Comment:
http://stackoverflow.com/questions/32418766/c-unresolved-external-symbol-sprintf-and-sscanf-in-visual-studio-2015

this procedure works for me finally.

add legacy_stdio_definitions.lib to uv.sln in vs2015 AND application.
libuv property in vs2015 -> change platform toolset v100(visual studio 2010) to v140(visaul studio 2015)
clean and rebuild both libuv and application.